server/lib/twitch.js

```diff
@@ -0,0 +1,64 @@
+const fetch = require('node-fetch');
+const WebSocket = require('ws');
+
+//ANDREW PLEASE
+function openSocketConnectionWithChannelId(channelId, oauth_token, callBack) {
+    ws = new WebSocket('wss://pubsub-edge.twitch.tv');
+
+    function heartbeat() {
+        message = {
+            type: 'PING',
+        };
+        console.log('SENT: ' + JSON.stringify(message));
+        ws.send(JSON.stringify(message));
```

Comment:
If this is just the stub, that's fine, but in the real implementation, I think we need dynamo.